src/index.ts

```diff
@@ -81,13 +81,14 @@ export const beautifier: Beautifier = {
       ],
       ternaryline: [
         ["multiline_ternary"],
-        function(options) {
-          if (options.multiline_ternary === "always") {
-            return false;
-          } else if (options.multiline_ternary === "never") {
-            return true;
-          } else {
-            return undefined;
+        (options): boolean | undefined => {
+          switch (options.multiline_ternary) {
+            case "always":
+              return false;
+            case "never":
+              return true;
+            default:
+              return undefined;
```

Comment:
I've been returning `undefined` with the expectation the base beautifier will simply NOT do anything with regards to this option. Hopefully Pretty Diff respects this, too. Prettier seemed to be OK with `undefined` option values.

Reply:
If it doesn't, I think we should return true for this particular option since that is prettydiff's default.